### ISSUE TITLE
Unparsable char: @reboot

### ISSUE THREAD

**Issue by shogochiai** [takumakanari/cronv#23]
# Error text
panic: Failed to analyze cron '@reboot root /bin/sleep 45;
/opt/codedeploy-agent/bin/update --sanity-check deb':
syntax error in minute field: '@reboot'

# Full file content
```
15 0 * * * root /bin/sleep 54; /opt/codedeploy-agent/bin/update --sanity-check --upgrade deb
45 * * * * root /bin/sleep 54; /opt/codedeploy-agent/bin/update --sanity-check --downgrade deb
@reboot root /bin/sleep 45; /opt/codedeploy-agent/bin/update --sanity-check deb
```

# Possible Solution
- Fix cron expression parser

**Comment by shogochiai** [comment]
I will note when I got new insight, I will fix when I cloud convince so.

**Comment by takumakanari** [comment]
Hi @shogochiai, thanks for your report.
I'll check it later.

**Comment by shogochiai** [comment]
Thanks for reply. Maybe this lines. I also check it.
https://github.com/takumakanari/cronv/blob/master/crontab.go#L46-L83

**Comment by takumakanari** [comment]
You're correct.
Also it func depends on `cronexpr`.

https://github.com/takumakanari/cronv/blob/master/ctx.go#L24

**Comment by takumakanari** [comment]
It seems that parsing `@reboot` is not supported in `cronexpr`, so I think there are 2 solutions to solve an issue:

1. Fix cron expression parser
    - solution is almost good 👍 
    - need to decide how to draw it schedule in the report html

2. Add option to skip invalid line
    - like `--skip-invalid-line`
    - just a workaround
    - someone may wants to see it schedule too, in the report html

**Comment by shogochiai** [comment]
Basically I vote for 2nd option, but we'd better to generate some annotation HTML that "Which lines are skipped."

---

FYI: According to `tkmgo/cronexpr`

<img width="987" alt="screenshot 2018-02-21 at 9 24 11 pm" src="https://user-images.githubusercontent.com/4891004/36485012-9f1c8b74-174d-11e8-824e-8fca01dbc1df.png">

The reason why `@reboot` is not supported: no equivalent expression.

We don't need to show it as time-sequential view, but we'd better to show proof of existence.

**Comment by takumakanari** [comment]
@shogochiai I almost completed #24, to include extra tasks in the report HTML like as follows:

![nv](https://user-images.githubusercontent.com/2722762/36538519-2e2a6f74-1817-11e8-9196-ef677231a027.png)

What do you think about this?

**Comment by takumakanari** [comment]
I'll merge #24 for now. 
We can open new issue or p-r if we need more fixes.

**Comment by shogochiai** [comment]
Awesome! Thanks for your quick response and improvement! I will test run on my environment as well.

**Comment by shogochiai** [comment]
Worked well when I fetched from current `master` branch.

I'm looking forward to see that it'll be released and everyone use it ;)

**Comment by takumakanari** [comment]
I just released `v0.3.0`. You can try it too.
